packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -947,24 +948,66 @@ abstract class ResidentRunner {
-        printStatus('To toggle the display of construction lines (debugPaintSizeEnabled), press "p".');
-        printStatus('To simulate different operating systems, (defaultTargetPlatform), press "o".');
-        printStatus('To toggle the elevation checker, press "z".');
+        printCommandHelp('L', 'Dump layer tree to the console.',
+            inParenthesis: 'debugDumpLayerTree');
+        printCommandHelp('S', 'Dump accessibility tree in traversal order.',
+            inParenthesis: 'debugDumpSemantics');
+        printCommandHelp(
+            'U', 'Dump accessibility tree in inverse hit test order.',
+            inParenthesis: 'debugDumpSemantics');
+        printCommandHelp('i', 'Toggle widget inspector.',
+            inParenthesis: 'WidgetsApp.showWidgetInspectorOverride');
+        printCommandHelp('p', 'Toggle the display of construction lines.',
+            inParenthesis: 'debugPaintSizeEnabled');
+        printCommandHelp('o', 'Simulate different operating systems.',
+            inParenthesis: 'defaultTargetPlatform');
+        printCommandHelp(
+            'z', 'Toggle elevation checker.');
       } else {
-        printStatus('To dump the accessibility tree (debugDumpSemantics), press "S" (for traversal order) or "U" (for inverse hit test order).');
+        printCommandHelp('S', 'Dump accessibility tree in traversal order.',
+            inParenthesis: 'debugDumpSemantics');
+        printCommandHelp(
+            'U', 'Dump accessibility tree in inverse hit test order.',
+            inParenthesis: 'debugDumpSemantics');
       }
-      printStatus('To display the performance overlay (WidgetsApp.showPerformanceOverlay), press "P".');
-      printStatus('To enable timeline events for all widget build methods, (debugProfileWidgetBuilds), press "a"');
-    }
-    if (flutterDevices.any((FlutterDevice d) => d.device.supportsScreenshot)) {
-      printStatus('To save a screenshot to flutter.png, press "s".');
+      printCommandHelp('P', 'Toggle performance overlay.',
+          inParenthesis: 'WidgetsApp.showPerformanceOverlay');
+      printCommandHelp(
+          'a', 'Toggle timeline events for all widget build methods.',
+          inParenthesis: 'debugProfileWidgetBuilds');
+    }
+  }
+
+  void printCommandHelp(String key, String description,
+      {String inParenthesis}) {
+    final String message = '${terminal.bolden(key)} $description';
+
+    String parentheticalText = '';
+    if (inParenthesis != null && inParenthesis.isNotEmpty) {
+      bool wrap = false;
+      int width = maxLineWidth - message.length;
+      parentheticalText = '($inParenthesis)';
+      if(width < parentheticalText.length) {
+        width = maxLineWidth;
+        wrap = true;
+      }
+      parentheticalText = (wrap ? '\n' : '') +
+          terminal.color(
+              parentheticalText, TerminalColor.grey)
+              .padLeft(width);
     }
+
+    printStatus(message + parentheticalText);
```

Comment:
you may wish to consider using a StringBuffer instead of the string concatenation. It may or may not make this clearer.